frame/dvm/src/mock.rs

```diff
@@ -152,6 +154,17 @@ impl AddressMapping<AccountId32> for HashedAddressMapping {
 	}
 }
 
+pub struct RingBack<T>(PhantomData<T>);
+impl<T: darwinia_evm::Config> Precompile for RingBack<T> {
+	fn execute(
+		input: &[u8],
+		target_gas: Option<u64>,
+		context: &Context,
+	) -> core::result::Result<(ExitSucceed, Vec<u8>, u64), ExitError> {
+		darwinia_evm_precompile_transfer::ring::RingBack::<T>::transfer(&input, target_gas, context)
+	}
+}
+
```

Comment:
Why can not use ```Transfer``` precompile for testing directly?

Reply:
We have discussed a lot in the dev slack channel concerning this isssue.  Those comments might explain why I do it this way.